Dockerfile

```diff
@@ -9,16 +11,15 @@ RUN apk update && \
 ## Populate app directory
 WORKDIR /home/gglsbl
 ENV GSB_DB_DIR /home/gglsbl/db
-COPY ["requirements.txt", "*.py", "logging.conf", "./"]
+COPY ["requirements.txt", "./"]
 ENV LOGGING_CONFIG /home/gglsbl/logging.conf
 
 # Install Python packages, cleanup and set permissions
-RUN pip3 install --upgrade pip setuptools && \
+RUN --mount=type=cache,target=/root/.cache \
```

Comment:
These extra parameters break building this on plain old Docker, please make sure that whatever changes you make to the Dockerfile keep backwards compability with `docker build`.

Reply:
Makes sense, I will update the PR